src/web/mod.rs

```diff
@@ -104,6 +111,7 @@ pub fn run(soup: Blender) -> HttpResult<Listening> {
     Server {
             handlers: router,
             host: 8080.into(),
+            global: Global::from(Box::new(Mutex::new(soup))),
```

Comment:
I guess it's fine to use a `Mutex` here given that we don't actually ever access it. Just feels dirty...